docs/what.rst

```diff
@@ -0,0 +1,23 @@
+=====================
+What is a Certificate?
+=====================
+
+An SSL *certificate* contains a public key and a private key that work together to enable
```

Comment:
I'm alarmed by a couple of things in this first sentence (!). First, it uses the popular term "SSL" which is obsolete and no longer technically correct. Second, the certificate doesn't "contain" the private key. (The certificate is public, but the private key is private, and in Certbot they're even kept in separate files.)